app/index.html

```diff
@@ -6,16 +6,19 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/bootstrap.css">
-    <script src="js/handlebars-v4.0.11.js"></script>
-    <script id="handle_bars" type="text/x-handlebars-template"></script>
-    <script src="js/userLogIn.js" defer></script>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" href="css/indexPage.css">
+    <script defer src="js/handlebars-v4.0.11.js"></script>
+    <script defer id="handle_bars" type="text/x-handlebars-template"></script>
+    <script defer src="js/userLogIn.js"></script>
     <title>Login page</title>
 </head>
 <body>
 <div class="container">
-    <div class="col-sm-12  text-center">
+    <img class="logo" alt="Mayden Logo" src="images/branding-icons/mayden_academy_logotype_2018_500px_colour.png">
+    <div class="col-sm-12 text-center">
         <div class="col-sm-offset-34col-sm-9 text-center">
-            <h3> Welcome to the Mayden Academy Aptitude Test </h3>
+            <h3 class="header"> Welcome to the Mayden Academy Aptitude Test </h3>
```

Comment:
quite generic class name. should hope that no one else will make this class name in other pages.